src/components/card/CardSidebar.vue

```diff
@@ -225,6 +225,9 @@ export default {
 		max-width: calc(100% - #{$modal-padding*2});
 		padding: 0 14px;
 		max-height: 100%;
+		user-select: text;
+		-webkit-user-select: text;
```

Comment:
What do you think about the support for other browsers?

-moz-user-select: text;
-webkit-user-select: text; 
-ms-user-select: text; 
user-select: text;

Reply:
All recent major browsers should be fine without the prefix except for safari, according to https://caniuse.com/mdn-css_properties_user-select ;)